Launcher/Classes/Extensions.swift

```diff
@@ -54,3 +54,12 @@ extension Bundle {
         return path(forResource: name, ofType: ext.rawValue)
     }
 }
+
+extension Dictionary where Key == String {
```

Comment:
Why should this only be used for a Dictionary with a key of `String`?

Reply:
Found it useful to kinda encapsulate it, as I expect this to apply to only this kind of Dictionaries